java/ql/lib/semmle/code/java/frameworks/Strings.qll

```diff
@@ -35,26 +44,43 @@ private class StringSummaryCsv extends SummaryModelCsv {
         "java.lang;String;false;toLowerCase;;;Argument[-1];ReturnValue;taint",
         "java.lang;String;false;toString;;;Argument[-1];ReturnValue;value",
         "java.lang;String;false;toUpperCase;;;Argument[-1];ReturnValue;taint",
+        "java.lang;String;false;transform;;;Argument[-1];Parameter[0] of Argument[0];value",
+        "java.lang;String;false;transform;;;ReturnValue of Argument[0];ReturnValue;value",
         "java.lang;String;false;translateEscapes;;;Argument[-1];ReturnValue;taint",
         "java.lang;String;false;trim;;;Argument[-1];ReturnValue;taint",
         "java.lang;String;false;valueOf;(char);;Argument[0];ReturnValue;taint",
         "java.lang;String;false;valueOf;(char[],int,int);;Argument[0];ReturnValue;taint",
         "java.lang;String;false;valueOf;(char[]);;Argument[0];ReturnValue;taint",
+        // TODO: Should `append` and `write` be modelled for Appendable and Writer instead?
+        // Could then remove some of the modelled `append` method here and for StringBuilder
```

Comment:
Should `Appendable` and `Writer` be modelled?

Reply:
This sounds like a positive step to me